frontend-react/src/components/Admin/EditReceiverSettings.tsx

```diff
@@ -0,0 +1,227 @@
+
+import {
+    CheckboxComponent,
+    TextAreaComponent,
+    TextInputComponent,
+} from "./AdminFormEdit";
+
+type Props = { orgname: string; receivername: string; action: string };
+
+export function EditReceiverSettings({ match }: RouteComponentProps<Props>) {
+    const orgname = match?.params?.orgname || "";
+    const receivername = match?.params?.receivername || "";
+    const action = match?.params?.action || "";
+    const history = useHistory();
+
+    const FormComponent = () => {
+        const orgReceiverSettings: OrgReceiverSettingsResource = useResource(
+            OrgReceiverSettingsResource.detail(),
+            { orgname, receivername, action }
+        );
+
+        const { fetch: fetchController } = useController();
+        const saveRecieverData = async () => {
+            switch (action) {
+                case "edit":
+                    try {
+                        const data = JSON.stringify(orgReceiverSettings);
+                        await fetchController(
+                            OrgReceiverSettingsResource.update(),
+                            { orgname, receivername: receivername },
+                            data
+                        );
+                        showAlertNotification(
+                            "success",
+                            `Item '${receivername}' has been updated`
+                        );
+                        history.goBack();
+                    } catch (e: any) {
+                        console.trace(e);
+                        showError(
+                            `Updating item '${receivername}' failed. ${e.toString()}`
+                        );
+                        return false;
+                    }
+                    break;
+                case "clone":
+                    try {
+                        const data = JSON.stringify(orgReceiverSettings);
+                        await fetchController(
+                            OrgReceiverSettingsResource.update(),
+                            { orgname, receivername: orgReceiverSettings.name },
+                            data
+                        );
+                        showAlertNotification(
+                            "success",
+                            `Item '${orgReceiverSettings.name}' has been created`
+                        );
+                        history.goBack();
+                    } catch (e: any) {
+                        console.trace(e);
+                        showError(
+                            `Cloning item '${
+                                orgReceiverSettings.name
+                            }' failed. ${e.toString()}`
+                        );
+                        return false;
+                    }
+                    break;
+                default:
+                    return false;
+            }
+
+            return true;
+        };
+
+        return (
+            <GridContainer containerSize={"desktop"}>
```

Comment:
This seems to be where the wrapping issue is for mobile. I was digging through TrussWorks' storybook and it looks like the column size and container size are independent of each other. Other libraries I've used allow for `sm`, `md`, `lg`, `xl` etc. props that let you say `when sm -> column size is 12. when md -> column size is 8`. I wonder if you can recreate that functionality.

Reply:
@kevinhaube gonna punt on this officially, but I _am_ going to play around with this a bit. Thank you!